mlir/include/mlir/IR/MLIRContext.h

```diff
@@ -153,6 +153,13 @@ class MLIRContext {
     disableMultithreading(!enable);
   }
 
+  /// Set the flag specifying if thread-local storage should be used
+  /// by storage allocators in this context.
```

Comment:
I'd note here that disabling multi-threading should also imply disabling thread local storage.